modules/virt-creating-custom-namespace-for-templates.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-creating-custom-namespace-for-templates.adoc
+
+[id="virt-creating-custom-namespace-for-templates_{context}"]
+= Creating a custom namespace for templates
+
+You can create a custom namespace that is used to deploy Red Hat templates for use by anyone who has permissions to access those templates.
```

Comment:
HI @sjhala-ccs - Users who use this content are expected to have gone through the content at this location:

https://docs.openshift.com/container-platform/4.9/virt/vm_templates/virt-creating-vm-template.html

There, we have a very detailed explanation of what we mean by Red Hat templates and how we provide 4 types of Red Hat templates to our users. 

SMEs have approved this content and we have had several meetings to discuss the deployment of Red Hat templates to a custom namespace. For instance, "user provided" templates are Red Hat templates that were cloned and created by users.

May I please request you to read: https://docs.openshift.com/container-platform/4.9/virt/vm_templates/virt-creating-vm-template.html

Please advise.

Reply:
Adding @sarahbx and @gouyang if they wish to add their comments here. Thanks.

Reply:
@lmandavi My question was if the term "Red Hat templates" applies to all 4 types of templates, especially since they have distinct names as seen in the linked topic. If that is what makes sense according to the SMEs, then I am fine with it. I actually suggested in [another comment](https://github.com/openshift/openshift-docs/pull/39535#discussion_r795798645) that you add an xref to the `virt-creating-vm-template.adoc` module so users have the resource handy.

Reply:
@sjhala-ccs - 1) In my understanding (and recollections in working with the developers who categorized Red Hat templates into four types), when we say Red Hat template, it could be any one of the 4 Red Hat templates. Since the SMEs approved the reference to Red Hat templates throughout this PR, my understanding is that this is an acceptable way of referring to virtual machine templates and we're aligned with how we defined Red Hat templates in the other content assembly 2) Adding an xref to the  virt-creating-vm-template.adoc module can be done. Will update you when that xref is completed.

Reply:
The distinction between the template types is important, as OpenShift Virtualization only manages VM templates. To be completely accurate, they are "Red Hat managed virtual machine templates", or "Virtual machine templates" if not referring to those managed by OpenShift Virtualization.

Any VM template may be deployed to the new custom namespace, but the primary focus of this documentation is removing the managed VM templates from the openshift namespace.

Reply:
@sarahbx - These are fine and granular distinctions but important ones. In the web console, we view and work with different types of Red Hat templates that are virtual machine templates. That was the reasoning for referring to the templates as Red Hat templates. In the context here, while working with the CLI to remove templates from the openshift namespace, you've both noted that the reference should be to virtual machine templates. I will update the doc accordingly to refer to virtual machine templates.